test/integration/base.py

```diff
@@ -333,6 +333,14 @@ def assertTableDoesNotExist(self, table):
             0
         )
 
+    def assertTableDoesExist(self, table):
+        columns = self.get_table_columns(table)
+
+        self.assertGreater(
+            len(columns),
+            0
+        )
```

Comment:
hmmmm ok. I'm going to merge this b/c the `already_exists` work is blocking. But I'll make an issue for better Snowflake integration tests